data/avgi/avgi ships.txt

```diff
@@ -0,0 +1,2142 @@
+	explode "large explosion" 12
+	"final explode" "final explosion medium"
+
+	description "The Frequentia-class Freighter is most commonly found serving as a fuel tender for ships in larger fleets, taking advantage of its large (and ungainly) cargo receptacle to store enough cryogenic deuterium fuel to satisfy a small fleet's needs."
+
+
+# Utility
+
+ship "Polari"
+	sprite "ship/avgi polari"
+	thumbnail "thumbnail/avgi polari"
+	swizzle 0
+	attributes
+		category "Utility"
+		licenses
+			"Avgi Civilian"
+		"shields" 600
+		"hull"	2100
+		"required crew" 6
+		"bunks" 12
+		"mass" 120
+		"drag" 1.5
+		"heat dissipation" 0.9
+		"fuel capacity" 1800
+		"cargo space" 42
+		"outfit space" 294
+		"weapon capacity" 36
+		"engine capacity" 60
+		"asteroid scan power" 72
+		"anchor point" 1
+		weapon
+			"blast radius" 90
+			"shield damage" 600
+			"hull damage" 360
+			"hit force" 480
+	outfits
+		"Phased Array Laser (Blue)"
+		"Phased Array Laser (Green)"
+
+		"Lantern Fission Core"
+		"Photoelectric Scales"
+		"Ultracapacitor Cell" 3
+		"Diffuse Deflector"
+		"Optical Lasing Generator"
+		"Multispectral Scanner" 3
+		"Spectrometer Array" 3
+		"Thermal Sink Partition"
+		"Deuterium Slush Tank"
+		"Electrostatic Ramscoop"
+		"Electromotive Rifle" 9
+
+		"Z-360 Fusion Torch"
+		"XY-180 Radial Thrusters"
+		"Hyperdrive"
+
+	turret -14 27 "Phased Array Laser (Green)"
+	turret 14 27 "Phased Array Laser (Blue)"
+
+	bay "Drone" -9 -25
+	bay "Drone" 9 -25
+	bay "Fighter" 0 48
+
+	engine 0 69
+	"reverse engine" -6 66
+		angle -30
+	"reverse engine" 6 66
+		angle 30
+	"steering engine" -3 -87
+		angle 90
+		right
+	"steering engine" 3 -87
+		angle -90
+		left
+	"steering engine" -6 66
+		angle 90
+		left
+	"steering engine" 6 66
+		angle -90
+		right
+
+	leak "leak" 30 30
+	leak "flame" 30 60
+	explode "tiny explosion" 12
+	explode "small explosion" 9
+	"final explode" "final explosion medium"
+
+	description "The Polari-class Scout is a legacy design that has managed to remain relevant in the current era by virtue of fulfilling a role historically not requiring much in the way of combat power. A specialized science ship, it is fitted with a spacious laboratory for analyzing samples and processing data, as well as an extensive suite of sensor arrays for recording observations. A pair of bays can deploy Optikon sensor drones, while an external mount is designed to hold an Akoustiki excursion vehicle for piloted sample gathering operations."
+	description "Desptie this versatility, the contested battlespace of the post-Incursion Avgi cluster is one in which the Polari is simply not as capable in. As science crews are increasingly asked to lead risky scouting expeditions back to the Avgi homeworld, demands for a newer, more survivable science ship routinely make their way to the Twilight Guard."
+
+
+ship "Entasi"
+	sprite "ship/avgi entasi"
+	thumbnail "thumbnail/avgi entasi"
+	swizzle 0
+	attributes
+		category "Utility"
+		licenses
+			"Avgi Civilian"
```

Comment:
```suggestion
		licenses
			"Avgi Civilian"
		"cost" 3360000
```
The combination of the decent bay count, high cargo space, decent weapon space and integrated `solar collection` make this quite a powerful ship, hence a higher cost than the other civilian utilities.